manager/job/sync_peers.go

```diff
@@ -175,84 +225,106 @@ func (s *syncPeers) createSyncPeers(ctx context.Context, scheduler models.Schedu
 
 // Merge sync peer results with the data in the peer table.
 func (s *syncPeers) mergePeers(ctx context.Context, scheduler models.Scheduler, results []*resource.Host, log *logger.SugaredLoggerOnWith) {
```

Comment:
Should we consider batch updating, creating, or deleting records for the merge operation to minimize database I/O times, especially when the number of peers is substantial?

Reply:
Yes, i use upsert instead of add and update. Check this out~